### ISSUE TITLE
Background and foreground colors have a sufficient contrast ratio

### ISSUE THREAD

**Issue by alij2016** [GoogleChrome/lighthouse#12349]
**Describe the bug**

Hi, I am pretty sure there is a bug regarding Background and foreground colors have a sufficient contrast ratio. My website, https://letsdiscovermauritius.com/about.html, has a black foreground and white background. After analyzing the contrast ratio, it is giving me 21, which is very good and valid. However, your system is detecting it as error. I see this on all pages with white background and text is black.

**Comment by alij2016** [comment]
So, is this clearly a bug then? On all my pages I got error.

**Comment by brendankenny** [comment]
This is #12039, which is waiting on an upstream fix from axe-core (https://github.com/dequelabs/axe-core/issues/2674). I'll dedupe into the existing issue.

**Comment by alij2016** [comment]
Why the thread is closed ?

**Comment by connorjclark** [comment]
Follow the other issue.
